raiden-dapp/src/views/TransferSteps.vue

```diff
@@ -250,6 +250,7 @@ export default class TransferSteps extends Mixins(
   pfsFeesPaid: boolean = false;
   pfsSelectionSkipped: boolean = false;
   routeSelectionSkipped: boolean = false;
+  paymentId: BigNumber = bigNumberify(randomBytes(8));
```

Comment:
nitpick: On Raiden-py if this is not defined the epoch in ms during the send is used instead. It is not required but maybe we could follow the same scheme. Is that ok?

Reply:
Shouldn't this be done in the SDK? I just chose the same method that was used there in case the `paymentId` is `undefined`

Reply:
I'd suggest leaving it as `undefined`, if it isn't provided. The SDK will create it then.

Reply:
I think @kelsos meant that the SDK should send the current timestamp if it's `undefined` (instead of random bytes currently)

Reply:
Yes, and that's what the SDK does, but only if it isn't handled an explicit `paymentId` (i.e. leaving it as undefined if isn't explicitly set)

Reply:
Have created #1187 for this

Reply:
Oh, right, the paymentId is random, the messageId is timestamp-based. I'd say this isn't critical, but there's a utility on the SDK which can be used on these cases, `makePaymentId`